hello-world/src/main/resources/archetype-resources/pom.xml

```diff
@@ -13,9 +13,6 @@
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
         <jenkins.version>2.7.3</jenkins.version>
-        <workflow-step-api-plugin.version>2.12</workflow-step-api-plugin.version>
-        <workflow-support-plugin.version>2.14</workflow-support-plugin.version>
-        <workflow-cps-plugin.version>2.39</workflow-cps-plugin.version>
```

Comment:
Only need properties if we are using plugin as well as `tests` artifact and need to align versions.